config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
```

Comment:
[ Optional] 
Kindly remove the commented code on line 6. You can always use git history to view deleted lines of code.